### ISSUE TITLE
Print preview-specific design

### ISSUE THREAD

**Issue by hbillings** [18F/cms-hitech-apd#1946]
The print preview works, but there's still lots of whitespace we can optimize away. Design should look at this to help come up with a consistent way to do these optimizations.

### This task is done when...
 - [x] The print preview has design that is specific to it, and is optimized for minimal whitespace and compactness, without sacrificing readability
 - [x] We have identified the key elements of the print preview necessary for quick scanning and wayfinding in a long document

**Comment by nkkl** [comment]
This is design-lab friendly.

**Comment by nkkl** [comment]
This will need more context to be design lab friendly; it would be ideal for that conversation to be documented here.

**Comment by jeromeleecms** [comment]
How scalable is our current solution? If there are 12 S&Cs and it changed in # (as it has in the past) is it easy to switch the data model to support a different number? If it's hard to make that happen, this consolidated format seems ideal as well.

**Comment by lauraponce** [comment]
Design lab request has been submitted: https://github.com/18F/design-lab/issues/97

**Comment by lauraponce** [comment]
xposting the design lab comment here since it's a private repo.

@jeromeleecms Igor posted an updated PDF with the new styles: 
[CMS.__.HITECH.__.APD_16.pdf](https://github.com/18F/cms-hitech-apd/files/3947702/CMS.__.HITECH.__.APD_16.pdf)

Here's what he shared for a summary of the changes:

1. I changed some of the hierarchy and spacing under Activity 1 as an example for the other sections. Within this section, I've made the headings that were previously a paragraph with a strong tag into an h4 with class ds-h4, and brought the paragraph beneath it closer to the heading. Similarly under "Federal Financial Participation (FFP) and Cost Allocation" I changed the title to an h3 and the subsequent content below is h4. I also made it so the the numbers were inline with their headings.

2. Updated the styling of the rules on the page so that major sections (h2s) have a thick black rule right above them, and subsections (h3s) have a thin grey rule.

3. Adjusted the styling of the running head (i.e. the current activity title)

4. Added margin space to the printed page
